art/attacks/poisoning/poisoning_attack_svm.py

```diff
@@ -55,7 +58,7 @@ class PoisoningAttackSVM(PoisoningAttackWhiteBox):
 
     def __init__(
         self,
-        classifier: "ScikitlearnSVC",
+        classifier: "SklearnClassifier",
```

Comment:
I'm not sure about this change because I think the attack is specific to SVM classifiers?

Reply:
You're right. I'll fix this in the next commit.